2022-05-05 22:20:41,227 INFO    MainThread:5237 [wandb_setup.py:_flush():75] Loading settings from /Users/zhangjessie/.config/wandb/settings
2022-05-05 22:20:41,227 INFO    MainThread:5237 [wandb_setup.py:_flush():75] Loading settings from /Users/zhangjessie/PycharmProjects/py3.7/spinup/algos/pytorch/sac/wandb/settings
2022-05-05 22:20:41,227 INFO    MainThread:5237 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-05-05 22:20:41,227 INFO    MainThread:5237 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'sac.py', 'program': '/Users/zhangjessie/PycharmProjects/py3.7/spinup/algos/pytorch/sac/sac.py'}
2022-05-05 22:20:41,227 INFO    MainThread:5237 [wandb_init.py:_log_setup():437] Logging user logs to /Users/zhangjessie/PycharmProjects/py3.7/spinup/algos/pytorch/sac/wandb/run-20220505_222041-klez0i6m/logs/debug.log
2022-05-05 22:20:41,227 INFO    MainThread:5237 [wandb_init.py:_log_setup():438] Logging internal logs to /Users/zhangjessie/PycharmProjects/py3.7/spinup/algos/pytorch/sac/wandb/run-20220505_222041-klez0i6m/logs/debug-internal.log
2022-05-05 22:20:41,227 INFO    MainThread:5237 [wandb_init.py:init():471] calling init triggers
2022-05-05 22:20:41,228 INFO    MainThread:5237 [wandb_init.py:init():475] wandb.init called with sweep_config: {}
config: {}
2022-05-05 22:20:41,228 INFO    MainThread:5237 [wandb_init.py:init():524] starting backend
2022-05-05 22:20:41,228 INFO    MainThread:5237 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-05-05 22:20:41,235 INFO    MainThread:5237 [backend.py:ensure_launched():217] starting backend process...
2022-05-05 22:20:41,243 INFO    MainThread:5237 [backend.py:ensure_launched():222] started backend process with pid: 5247
2022-05-05 22:20:41,246 INFO    MainThread:5237 [wandb_init.py:init():533] backend started and connected
2022-05-05 22:20:41,256 INFO    MainThread:5237 [wandb_init.py:init():597] updated telemetry
2022-05-05 22:20:41,258 INFO    MainThread:5237 [wandb_init.py:init():628] communicating run to backend with 30 second timeout
2022-05-05 22:20:42,020 INFO    MainThread:5237 [wandb_run.py:_on_init():1923] communicating current version
2022-05-05 22:20:42,145 INFO    MainThread:5237 [wandb_run.py:_on_init():1927] got version response upgrade_message: "wandb version 0.12.16 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-05-05 22:20:42,145 INFO    MainThread:5237 [wandb_init.py:init():659] starting run threads in backend
2022-05-05 22:20:43,547 INFO    MainThread:5237 [wandb_run.py:_console_start():1897] atexit reg
2022-05-05 22:20:43,548 INFO    MainThread:5237 [wandb_run.py:_redirect():1770] redirect: SettingsConsole.REDIRECT
2022-05-05 22:20:43,548 INFO    MainThread:5237 [wandb_run.py:_redirect():1775] Redirecting console.
2022-05-05 22:20:43,550 INFO    MainThread:5237 [wandb_run.py:_redirect():1831] Redirects installed.
2022-05-05 22:20:43,551 INFO    MainThread:5237 [wandb_init.py:init():684] run started, returning control to user process
2022-05-05 22:22:58,237 INFO    MainThread:5237 [wandb_run.py:_atexit_cleanup():1866] got exitcode: 255
2022-05-05 22:22:58,238 INFO    MainThread:5237 [wandb_run.py:_restore():1838] restore
2022-05-05 22:23:00,764 INFO    MainThread:5237 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 531
  total_bytes: 531
}

2022-05-05 22:23:00,953 INFO    MainThread:5237 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 531
  total_bytes: 531
}

2022-05-05 22:23:01,435 INFO    MainThread:5237 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 531
  total_bytes: 7678
}

2022-05-05 22:23:01,542 INFO    MainThread:5237 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 531
  total_bytes: 7678
}

2022-05-05 22:23:01,645 INFO    MainThread:5237 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7678
  total_bytes: 7678
}

2022-05-05 22:23:01,749 INFO    MainThread:5237 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7678
  total_bytes: 7678
}

2022-05-05 22:23:01,856 INFO    MainThread:5237 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7678
  total_bytes: 7678
}

2022-05-05 22:23:01,962 INFO    MainThread:5237 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7678
  total_bytes: 7678
}

2022-05-05 22:23:02,279 INFO    MainThread:5237 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7678
  total_bytes: 7678
}

2022-05-05 22:23:02,565 INFO    MainThread:5237 [wandb_run.py:_on_finish():1995] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7678
  total_bytes: 7678
}
local_info {
}

2022-05-05 22:23:03,949 INFO    MainThread:5237 [wandb_run.py:_footer_history_summary_info():3102] rendering history
2022-05-05 22:23:03,949 INFO    MainThread:5237 [wandb_run.py:_footer_history_summary_info():3134] rendering summary
2022-05-05 22:23:03,951 INFO    MainThread:5237 [wandb_run.py:_footer_sync_info():3057] logging synced files
